style.css

```diff
@@ -1 +1,285 @@
-/*  Add your styling here */
+body {
+    border: 3px solid black;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 0.3fr 0.3fr 0.5fr 0.5fr 0.5fr 0.5fr 0.5fr;
+    grid-gap: 2px;
+
+    grid-template-areas:
+        "i bor"
+        "i pa"
+        "wt wt"
+        "ar ar"
+        "mimg mimg"
+        "f1 f2"
+        "f3 f4"
+        "fo fo"
+    ;
+
+}
+
+
+.icon_img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-area: i;
+    
+
+}
+
+.bor {
+    grid-area: bor;
+    text-align: end;
+    font-size: 15px;
+    
+}
+
+
+
+.p_nav {
+    margin: 0;
+    display: flex;
+    text-align: end;
+    justify-content: end;
+    color: rgb(40, 61, 54);
+    grid-area: pa;
+    font-size: 13px;
+}
+
+
+.wel_tex {
+    grid-area: wt;
+    text-align: start;
+    font-size: 30px;
+    background-color: rgb(114, 146, 235);
+}
+
+article {
+    grid-area: ar;
+}
+
+.main_img {
+    grid-area: mimg;
+    height: 100px;
+}
+
+.fimg_pic {
+    grid-area: f1;
+    height: 80px;
+}
+
+.simg_pic {
+    grid-area: f2;
+    height: 80px;
+}
+
+.timg_pic {
+    grid-area: f3;
+    height: 80px;
+}
+
+.fourth_pic {
+    grid-area: f4;
+    height: 80px;
+}
+
+.footer {
+    grid-area: fo;
+    background-color: rgb(239, 143, 70);
+}
+
+.join_us {
+    text-align: center;
+}
+
+.footer_imgs {
+    display: flex;
+    justify-content: center;
+    justify-content: space-between;
+}
+
+.footer_ico {
+    display: flex;
+    width: 30px;
+    height: 20px;
+    border-radius: 70px;
+    padding: 8px;
+
+}
+
+ul {
+    list-style: none;
+}
+
+@media screen and (min-width: 540px) {
+    body {
+        background-color: rgb(202, 164, 206);
+
+    }
+
+    .box {
+        padding: 21px;
+    }
+
+    .container {
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+        grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+        grid-area:
```

Comment:
`grid-area` is for assigning an element to a particular area within a grid, were you thinking of `grid-template-areas` here? Like you did [here](https://github.com/CodeYourFuture/HTML-CSS-Coursework-Week3/pull/365/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR11)?

This actually isn't causing any problems, because the browser will discard the property as it doesn't have a valid value, but you can remove this for tidiness.

Hint: you can see when a CSS property isn't valid in the browser's dev tools:

![image](https://user-images.githubusercontent.com/1737376/185490690-2901f4c0-c8ab-4b70-9279-795c438f381c.png)